bin/dungeon-revealer

```diff
@@ -1,42 +1,44 @@
 #!/usr/bin/env node
-var debug = require('debug')('dungeon-revealer');
```

Comment:
We probably should run this through a linter.

Reply:
Yeah, I realized that after I wrote `"lint": "./node_modules/.bin/eslint \"./public/javascripts/*.js\"",` in package.json. @n1ru4l added an eslint exclude that would exclude the node_modules anyway, so there is no risk in linting **/*.js I believe.

Reply:
Will fix this after I rebased the `master` branch